### PR TITLE
Little fix in the setup.py file, for more readable, also add .idea fo…

### PR DESCRIPTION
**Pull request by MichaelYusko**
…lder into .gitignore file

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -50,7 +50,10 @@ def find_packages(where='.', lib_prefix='', exclude=()):
     url='https://github.com/mozilla/jx-sqlite',
     license='MPL 2.0',
     packages=find_packages('.', lib_prefix=''),
-    install_requires=["future", "mo-collections>=1.2.17235", "mo-dots>=1.5.17188", "mo-files>=1.2", "mo-json-config", "mo-json>=1.0.17168", "mo-kwargs", "mo-logs", "mo-math", "mo-testing>=1.0.17168", "mo-threads", "mo-times"],
+    install_requires=["future", "mo-collections>=1.2.17235", "mo-dots>=1.5.17188",
+                      "mo-files>=1.2", "mo-json-config", "mo-json>=1.0.17168",
+                      "mo-kwargs", "mo-logs", "mo-math", "mo-testing>=1.0.17168",
+                      "mo-threads", "mo-times"],
```
Comment (jmaher) [inline setup.py]: could you make these one requirement per line:
install-requires=["future",
                            "mo-collections>=1.2.17235",
                           ...],

File: setup.py [hunk 2]
```diff
@@ -25,18 +25,18 @@ def find_packages(where='.', lib_prefix='', exclude=()):
...
-        where,prefix = stack.pop(0)
+        where, prefix = stack.pop(0)
         for name in os.listdir(where):
             fn = os.path.join(where,name)
             if ('.' not in name and os.path.isdir(fn) and
-                os.path.isfile(os.path.join(fn,'__init__.py'))
+                os.path.isfile(os.path.join(fn, '__init__.py'))
             ):
-                out.append(prefix+name); stack.append((fn,prefix+name+'.'))
+                out.append(prefix+name); stack.append((fn, prefix+name+'.'))
```
Comment (jmaher) [inline setup.py]: there are string concat with '+', can you add a space around the +, so we have ' + ' ?

Comment (MichaelYusko) [reply setup.py]: Yeah, no problem

File: setup.py (hunk 1 above)
Comment (MichaelYusko) [reply setup.py]: Will do it.

**Comment by jmaher** [comment]
Thanks for the cleanup here @MichaelYusko, I think these changes look great.  @klahnakoski can you merge this?

**Comment by MichaelYusko** [comment]
@jmaher No problem;)

**Pull request merged**